test/gui/tst_checkAlltabs/test.feature

```diff
@@ -14,7 +14,7 @@ Feature: Visually check all tabs
             | Settings     |
             | QuitOwncloud |
 
-
+    @skip
```

Comment:
Squish image `squish:fedora-38` has a problem showing desktop-ui with the drone agent os (i.e. Ubuntu 20.04).

The background is all black:

![Open_log_dialog_with_Ctrl+l_keys_combination](https://github.com/owncloud/client/assets/52366632/2f2380e4-ef17-43e6-8042-7336f9e2580b)